### PR TITLE
Attendance tracker

### PR DESCRIPTION
**Pull request by nayakastha**

### CODE REVIEW

**Comment by DesignrKnight** [comment]
@nayakastha 
Can you  scp/.flutter-plugins-dependencies delete it and add to gitignore

**Comment by DesignrKnight** [comment]
Hey @nayakastha , can you edit and resolve conflicts, or should I?

**Comment by nayakastha** [comment]
Can you do it. I am pretty  much confused what I did with the .flutter-plugins-dependencies

**Comment by nayakastha** [comment]
Hey @DesignrKnight , I made all the changes required. Everything in functioning well.

- [X] The subject list shows all the subjects of the section. (different list for autumn and spring semester)

- [X] A section selector is attached.

- [X] The screen overflow has been managed.

still there are some conflicts. Would you please take a look at it? I am unable to resolve

**Comment by DesignrKnight** [comment]
I'll fix the merge conflicts
Did you update route as per new routes by @SriramPatibanda ?

**Comment by nayakastha** [comment]
Yup I did us that.

**Comment by nayakastha** [comment]
Yup, I did use that.

File: scp/lib/time_table.dart [hunk 1]
```diff
@@ -4,6 +4,10 @@ import 'package:shared_preferences/shared_preferences.dart';
 import 'time_table_resources.dart';
 import 'package:url_launcher/url_launcher.dart';
 
+String theorySection = 'E';
```
Comment (ankank30) [inline scp/lib/time_table.dart:7]: Why do we need these variables outside TimeTableState?

**Review by ankank30 — APPROVED** [review]
Looks good but do reply to the comment before merging.

File: scp/lib/time_table.dart (hunk 1 above)
Comment (nayakastha) [reply scp/lib/time_table.dart:7]: I had changed it for accessing from another file. I guess it's not needed again so I will revert it back.

**Review by DesignrKnight — APPROVED** [review]

**Comment by DesignrKnight** [comment]
### Hacktoberfest Thank You from NIT Rourkela community
It was a super fun experience in Hacktoberfest with you. We hope you also had fun and amazing learning. We look forward to you in our upcoming sessions and events as part of the community. 
You can join our Slack community at http://bit.ly/NITRDevs

**As a token of our gratitude, we have set up digital badges of recognition for our contributors from Hacktoberfest. 
Do claim yours at https://live.dscnitrourkela.org/**

Thanks and Cheers!
Stay Safe

**Pull request merged**